stable/mission-control/values.yaml

```diff
@@ -307,6 +307,30 @@ elasticsearch:
   #    memory: "2.5Gi"
   #    cpu: "500m"
 
+  livenessProbe:
+    enabled: true
+    config: |
+      httpGet:
+        path: /_cat/health
+        port: {{ .Values.elasticsearch.httpPort }}
+      initialDelaySeconds: 90
+      periodSeconds: 10
+      timeoutSeconds: 10
+      failureThreshold: 3
+      successThreshold: 1
+
+  readinessProbe:
+    enabled: true
+    config: |
+      httpGet:
+        path: /_cat/health
+        port: {{ .Values.elasticsearch.httpPort }}
+      initialDelaySeconds: 60
```

Comment:
It doesn't make sense that the liveness probe has a shorter `initialDelaySeconds` than the readiness probe. They should either be the same or it should be the other way around

Reply:
`initialDelaySeconds` for liveness probe is 90 seconds 
and for readiness is 60 seconds .